narwhals/_pandas_like/expr.py

```diff
@@ -309,6 +309,15 @@ def ends_with(self, suffix: str) -> PandasExpr:
             suffix,
         )
 
+    def contains(self, pattern: str, *, literal: bool = False) -> PandasExpr:
```

Comment:
let's try to not set defaults in `_pandas_like`, so we make sure they get passed down explicitly from above
```suggestion
    def contains(self, pattern: str, *, literal: bool) -> PandasExpr:
```

Reply:
Alright